docs/manual-CN/1.overview/2.quick-start/2.FAQ.md

```diff
@@ -149,6 +150,12 @@ nebula> SHOW CONFIGS;
 
 [[↑] 回到顶部](#常见问题)
 
+### 配置启动不生效
+
+改了配置启动不生效，因为 **Nebula Graph** 默认从 meta 获取配置，在配置文件 `metad.conf`、`storaged.conf`、`graphd.conf` 中分别添加 `local_config=true`，从本地配置文件获取。详情参考[服务器配置](../../3.build-develop-and-administration/3.deploy-and-administrations/server-administration/configuration-statements/configs-syntax.md)。
```

Comment:
Do we need to specify the specific place to add the command `local_config=true`  and give more details about the directory of the `metad.conf`、`storaged.conf`、`graphd.conf` files?

Reply:
no, but I'll show the files directory.